components/cards.css

```diff
@@ -0,0 +1,213 @@
+:root {
+    --badge-background: #e0e0e03b;
+    --badge-color-green: #3dd33d;
+    --size-active: #555353;
+    --badge-background-overlay: #eaeaec99;
+    --box-shadow-card: rgba(0, 0, 0, 0.25) 0px 0.0625em 0.0625em, rgba(0, 0, 0, 0.25) 0px 0.125em 0.5em, rgba(255, 255, 255, 0.1) 0px 0px 0px 1px inset;
+}
+
+.card-badge {
+    background-color: var(--badge-background);
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 18rem;
+    height: max-content;
+    padding: 0.8rem;
+    border-radius: 8px;
+}
+
+.product {
+    border-radius: 8px;
+}
+
+.prod-container {
+    position: relative;
+}
+
+.prod-container img {
+    max-width: 100%;
+    height: auto;
```

Comment:
Typo: Maybe here you wanted to use max-height property...

Reply:
No. This is for the responsive image. I think max-width and height auto go together right?

Reply:
I was saying that you are using height property two times with different values _height: auto;_  and   _height: 100%;_